_includes/footer.html

```diff
@@ -31,18 +31,28 @@
       <div class="suggestion-text">{{{text}}}</div>
     </div>
   </div>
-  {% endraw %}
+  {% endraw %}-
```

Comment:
This is from the older PR. Extra `-` symbol

Reply:
thx